plugins/humanitec-backend-scaffolder-module/dist-dynamic/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@humanitec/backstage-plugin-scaffolder-backend-module-dynamic",
```

Comment:
@davidfestal is it required to publish additional modules or could the information also be embedded into the existing ones?